tests/acceptance/docker/src/run-tests.sh

```diff
@@ -9,6 +9,7 @@ git config --global advice.detachedHead false
 
 if [ "$TEST_SOURCE" = "oc10" ]
 then
+    export ACCEPTANCE_TEST_TYPE='core-api'
```

Comment:
what uses this code path?
do we still have test runs that have `"$TEST_SOURCE" = "oc10"` ?

Reply:
Probably this whole `"$TEST_SOURCE" = "oc10" ` "if" block is not used any more?
I suppose it was used for parallel deployment testing, and I think that we have removed the code for that.
But we can clean up old oarallel deployment and code that tries to mention "oc10" in another cleanup PR. That is not really related to the refactoring work of this PR.

Reply:
> Probably this whole `"$TEST_SOURCE" = "oc10" ` "if" block is not used any more? I suppose it was used for parallel deployment testing, and I think that we have removed the code for that. But we can clean up old oarallel deployment and code that tries to mention "oc10" in another cleanup PR. That is not really related to the refactoring work of this PR.

I can to this clean up of unnecessary code in my PR: https://github.com/owncloud/ocis/pull/9900